design/sketch.webidl

```diff
@@ -481,6 +481,14 @@ interface WebGPUProgrammablePassEncoder {
 interface WebGPURenderPassEncoder : WebGPUProgrammablePassEncoder {
     void setBlendColor(float r, float g, float b, float a);
     void setStencilReference(u32 reference);
+
+    // The default viewport is (0.0, 0.0, w, h, 0.0, 1.0), where w and h are the dimensions of back buffer
+    void setViewport(float x, float y, float, width, float height, float minDepth, float maxDepth);
```

Comment:
Rouge comma (`,`) between `float width`. :)

Reply:
Fixed in prep for merge. Thanks for finding this!

Reply:
Thanks!